pom.xml

```diff
@@ -31,7 +31,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- version properties -->
     <avro.version>1.7.7</avro.version>
-    <cdap.version>6.2.0</cdap.version>
+    <cdap.version>6.4.0-SNAPSHOT</cdap.version>
```

Comment:
Is this change mandatory? Why we have to change CDAP version in pom.xml?

Reply:
> Why do we have to change the version of CDAP in pom.xml?

Plugin has ETL Test, but tests are ignored if there are no Salesforce Account Credentials set in environment variable.  https://github.com/data-integrations/salesforce/blob/f3d38b2470f399a1869e7ad3129796b0dcadf73e/src/test/java/io/cdap/plugin/salesforce/etl/BaseSalesforceETLTest.java#L47-L60

Reply:
@rmstar  

Prashant, can you give us a hand on this matter? 

Is changing the cdap version in pom.xml the right approach to handle this scenario?

Reply:
Yes, if you have a dependency on an API that was added in 6.4.0 (going to be released in a couple of weeks) then you need to update cdap version in pom.xml.
Once 6.4.0 is released I would suggest changing cdap version from 6.4.0-SNAPSHOT to 6.4.0. Prefer not to depend on snapshot version.

Reply:
Can you clarify what change was made in this PR that requires CDAP 6.4.0 dependency? I'd like to avoid bumping up the version of CDAP dependency if possible.

Reply:
There was no specific requirement in the PR that required this dependency change but when we added ETL Tests we faced this issue. 

To verify this, we did the following: 

1. We run the ETL Tests from the develop branch without our changes, and we encountered the same issue. 
2. When we checked out the history from git, the minimum version of CDAP was changed to 6.4.0 but we think that after this change the ETL tests were skipped because you need to provide Salesforce account credentials. 

You can verify this by checking this out:
https://github.com/data-integrations/salesforce/blob/08b5980eec121c23a4529a5d5bcf805c5003776c/src/test/java/io/cdap/plugin/salesforce/etl/BaseSalesforceETLTest.java#L81-L89

Please let us know if there's any other way to run only the ETL Tests in CDAP 6.4.0.

Reply:
Chatted with @chtyim about this - on develop branch there is a dependency on CDAP 6.4.0 introduced by recent oauth changes. 
However, on release/1.3 branch we don't need to bump up the dependency.
So let's do the following:
1. Merge this PR with CDAP depedency version set to 6.4.0.
2. When you backport this PR to release/1.3 branch, set cdap.version = 6.2.0.